### ISSUE TITLE
Limit slides in deck to a class of section

### ISSUE THREAD

**Issue by ljpaul** [hakimel/reveal.js#1059]
Utilizing RevealJS for a lobby slideshow. Would be great if there was an option to limit the current slideshow to a class of section elements so that slides can be dynamically added and removed from the deck that is displayed.

**Comment by Delapouite** [comment]
Yes, Reveal could decide to exclude slides with an explicit `hidden` attribute : 
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden

**Comment by demoneaux** [comment]
@hakimel opinions? This also seems a bit of an edge case to me, you could do something like modify/remove elements from the DOM yourself and then `Reveal.sync()`.
